src/ProjectRequirements.php

```diff
@@ -23,6 +23,8 @@ class ProjectRequirements extends RequirementCollection
     const REQUIRED_PHP_VERSION_3x = '5.5.9';
     const REQUIRED_PHP_VERSION_4x = '7.1.3';
     const REQUIRED_PHP_VERSION_5x = '7.2.9';
+    const REQUIRED_PHP_VERSION_6x = '8.1.0';
```

Comment:
This is tricky as we changed the min version from 6.0 to 6.4. Here, I'm using the 6.4 min PHP version, which seems reasonnable.

Reply:
i discovered the project dir is off, when the requirements checker is globally installed:

>  * PHP version must be at least 8.2.0 (8.1.30 installed)

the resolved root dir is a global one in our case :sweat_smile: so the http-kernel version cannot be resolved.

ideally we'd tell the requirents checker the project root dir (`/app`)

this is not a big deal for us though, since the sf/php update is pending

Reply:
never mind, we can specify the project dir using args already!

https://github.com/symfony/requirements-checker/blob/e158a5ba12e1b84e2086a9a34ff9b63b03c5c1ed/bin/requirements-checker.php#L43